dhall/src/Dhall/Repl.hs

```diff
@@ -579,9 +580,9 @@ greeter =
   in liftIO (putStrLn message)
 
 
-dontCrash :: ( MonadIO m, Haskeline.MonadException m ) => m () -> m ()
+dontCrash :: ( MonadCatch m, MonadIO m ) => m () -> m ()
 dontCrash m =
-  Haskeline.catch
+  Control.Monad.Catch.catch
```

Comment:
I don't have a great understanding of exceptions in Haskell, so I'm wondering whether there's anything that I should look at closely here…

Reply:
@sjakobi: This should be fine.  It's basically the same thing as `Control.Exception.catch`